install/installer/setup_ssh_key

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
This is is similar to our backup-masters file which is also empty and does no harm but it helps users to know that they can modify this file if they require DcsMaster to start on a specific node. Dcs documentation will be updated in the subsequent PR